tests/t_libtest.cpp

```diff
@@ -5,7 +5,7 @@
 
 using namespace regexbench;
 
-static void report(const std::map<std::string, size_t>& m)
+void report(const std::map<std::string, size_t>& m, void* p)
```

Comment:
```
void report(const std::map<std::string, size_t>& m, void*)
```
instead of
```
void report(const std::map<std::string, size_t>& m, void* p)
```